Web_Development/FrontEnd/HTML_IFrames/iframes.md

```diff
@@ -5,8 +5,11 @@ An IFrame (Inline Frame) is an HTML document embedded inside another HTML docume
 The IFrame HTML element is often used to insert content from another source,such as an advertisement, into a Web page.
 
 (Basically it is used to display a web page within a web page)
+
```

Comment:
Can you add you add some more points explaining Iframes?the info is so little

Reply:
is it fine now @ghulamyazdani ?